src/app/components/layout/PrivateLayout.js

```diff
@@ -43,11 +43,11 @@ const PrivateLayout = ({ children, location }) => {
                 <PrivateHeader expanded={expanded} onToggleExpand={() => setExpand(!expanded)} />
                 <div className="flex flex-nowrap">
                     <Route path="/:path" render={() => <Sidebar url="/account" expanded={expanded} list={list} />} />
-                    <div className="main flex-item-fluid main-area scroll-smooth-touch" ref={mainAreaRef}>
+                    <MainContainer rootRef={mainAreaRef}>
```

Comment:
For now I would prefer doing

```js
mainAreaRef.current.scrollTop = 0;
```

at this line
https://github.com/ProtonMail/proton-vpn-settings/blob/master/src/app/components/layout/PrivateLayout.js#L37

in each application that needs it

instead of extracting the main component and doing it there

because this structure will be changed eventually